Assets/_Application/Layers/LayerTypes/Polygons/ObjectScatterLayerGameObject.cs

```diff
@@ -362,6 +355,138 @@ public void AddToProperties(RectTransform properties)
             propertySection.Settings = settings;
         }
 
+        public void LoadProperties(List<LayerPropertyData> properties)
+        {
+            var scatterSettings = (ScatterGenerationSettingsPropertyData)properties.FirstOrDefault(p => p is ScatterGenerationSettingsPropertyData);
```

Comment:
Hier hebben we tegenwoordig de `properties.Get<ScatterGenerationSettingsPropertyData>` methode voor, die een beetje werkt als GetComponent :) Niet fout, meer een leuke toevoeging